### PR TITLE
Feature/point visualization

### PR DESCRIPTION
**Pull request by nd-novorender**
https://trello.com/c/7tOCadxo/860-classification-and-other-pointcloud-visualization-updates

Test scenes are in task description.

- Add "Point visualization" widget
- Change icon for "Deviations" widget
- Show stamp for Classification and Intensity modes (both covered by ClassificationStamp) similar to DeviationStamp (only in ortho)

### CODE REVIEW

File: src/features/pointVisualization/routes/csvImportView.tsx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    };
+    const history = useHistory();
+    const dispatch = useAppDispatch();
+
+    const columns = useMemo(() => {
+        if (kind === "classification") {
+            return [
+                { name: "position", label: "Code", type: ColumnType.Number, required: true },
+                { name: "label", label: "Label", type: ColumnType.String },
+                { name: "color", label: "Color", type: ColumnType.Color, required: true },
```
Comment (barabadzhi) [inline src/features/pointVisualization/routes/csvImportView.tsx:44]: translate labels?

File: src/features/pointVisualization/routes/csvImportView.tsx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+            ];
+        } else {
+            return [
+                { name: "position", label: "Elevation", type: ColumnType.Number, required: true },
+                { name: "color", label: "Color", type: ColumnType.Color, required: true },
```
Comment (barabadzhi) [inline src/features/pointVisualization/routes/csvImportView.tsx:49]: same

**Review by barabadzhi — CHANGES_REQUESTED** [review]

**Comment by nd-novorender** [comment]
@barabadzhi all resolved, thanks

File: src/features/pointVisualization/components/header.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+        <>
+            <Box boxShadow={theme.customShadows.widgetHeader}>
+                <Box px={1}>
+                    <Divider />
+                </Box>
+                <Box display="flex" justifyContent="flex-end">
+                    {(kind === "classification" || kind === "elevation") && (
+                        <Button component="label" color="grey" tabIndex={-1} startIcon={<CloudUpload />}>
+                            {t("import")}
+                            <VisuallyHiddenInput type="file" onChange={importCsv} accept=".csv" />
```
Comment (barabadzhi) [inline src/features/pointVisualization/components/header.tsx:112]: `VisuallyHiddenInput` is not really needed, use `<input hidden>`, should be enough?
[Ref](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/hidden)

File: src/features/pointVisualization/routes/csvImportView.tsx [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+            ];
+        } else {
+            return [
+                { name: "position", label: t("elevation"), type: ColumnType.Number, required: true },
+                { name: "color", label: t("color"), type: ColumnType.Color, required: true },
+            ];
+        }
+    }, [t, kind]);
+
+    const [mapping, setMapping] = useState(csv.data[0].map(() => ""));
```
Comment (barabadzhi) [inline src/features/pointVisualization/routes/csvImportView.tsx:54]: Looks like we don't do any validations..
Does the format (data exported in CSV) have a header? If no, can it (optional)?
The default papa parsing seems to ignore the header. Is it what we alwaysget for the uploaded files?

Comment (nd-novorender) [reply src/features/pointVisualization/routes/csvImportView.tsx:54]: We skip all rows where color or position are invalid, so header will be skipped together with any invalid data.

**Review by barabadzhi — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**